modules/customizing-the-global-header/proc-customizing-rhdh-global-header.adoc

```diff
@@ -0,0 +1,38 @@
+[id="customizing-your-product-global-header_{context}"]
+= Customizing your {product} global header
+
+To replace the {product} global header with a custom header of choice, disable the default header.
+
+.Prerequisites
+* You have configured your {product-custom-resource-type} instance.
+* You have configured the support URL in the `app-config.yaml` file.
+* You have installed the notification plugin.
+
+.Procedure
+
+* Install your own header as a dynamic plugin as shown in the following example:
+
+[source,yaml,subs="+attributes,+quotes"]
+----
+# Custom header implementation
+- package: <npm or oci package-reference>
+  disabled: false
+  pluginConfig:
+    dynamicPlugins:
+      frontend:
+        <package_name>:
+          mountPoints:
+            - mountPoint: application/header
+              importName: <Header component name>
+              config:
+                position: above-main-content
+----
+
+* To disable the global header, disable the right plugin in your `app-config.yaml` configuration file as follows:
+
+[source,yaml,subs="+attributes,+quotes"]
+----
+# Disabling global header
+- package: ./dynamic-plugins/dist/red-hat-developer-hub-backstage-plugin-global-header
+  disabled: true
```

Comment:
Can you verify the content goes into the `app-config.yaml` file and not into `dynamic-plugins.yaml`? 

The content looks like content we are used to see in the `dynamic-plugins.yaml` but not in the `app-config.yaml` file. See `dynamic-plugins.yaml` examples in See https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.4/html/installing_and_viewing_plugins_in_red_hat_developer_hub/rhdh-installing-rhdh-plugins_title-plugins-rhdh-about#proc-config-dynamic-plugins-rhdh-operator_rhdh-installing-rhdh-plugins